scripts/redact_cli_py/instruction.md

```diff
@@ -0,0 +1,102 @@
+
+## Language support
+This tool supports Latin characters redaction only. For any non-Latin document support, please [contact us](mailto:formrecog_contact@microsoft.com?subject=Redaction%20tool%20language%20support).
+
+## Prerequisites
+- Azure subscription - [Create one for free](https://azure.microsoft.com/free/cognitive-services)
+- Once you have your Azure subscription, create a [Form Recognizer resource](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesFormRecognizer) in the Azure portal to get your key and endpoint.
+- Create an Azure storage account with a container - following the [Azure Storage quickstart for Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal). Use the *standard* performance tier.
+- [Optional] If the data cannot leave your environment, you need to follow [this document](https://docs.microsoft.com/en-us/azure/applied-ai-services/form-recognizer/label-tool#set-up-the-sample-labeling-tool) to set up the labeling tool container.
+
+## Step 1 - label the PII/sensitive fields with Sample Labeling Tool (FOTT)
+* Assemble your raw images based on the input requirements below and [upload them into a blob storage container](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/build-training-data-set#upload-your-training-data). Single-page PDF is supported for batch redaction only. You need to convert multi-page PDF files to images with below format before labeling.
+  - Format must be JPG, JPEG, PNG, BMP or TIFF.
+  - File size must be less than 50 MB.
+  - Image dimensions must be between 50 x 50 pixels and 10000 x 10000 pixels.
+* Generate a Shared access (SAS) token and URL for FoTT to access the blob storage. Make sure you have selected all the options in “Permissions”.
+![generate-sas-token](./images/SAS-token.png)
+* Go to [FoTT portal](https://fott-2-1.azurewebsites.net/) and create a new data connection by clicking on the left side navigation icon.
+![fott-connection](./images/fott-connection.png)
+* Got to the homepage of FoTT portal and create a new project by clicking “Use Custom to train a model with labels and get key value pairs”.
+![fott-project](./images/fott-project.png)
+* Once your project is created and opened, “Run layout on all documents” to get the OCR results with yellow bounding box first. This will make your tagging easier in later steps.
+![fott-run-layout](./images/fott-run-layout.png)
+* Label the PII or sensitive fields only by following [this guidance](https://docs.microsoft.com/en-us/azure/applied-ai-services/form-recognizer/label-tool?tabs=v2-1#label-your-forms). Please try to leverage the OCR detected results (aka. yellow bounding boxes as much as possible). You can multi-select these bounding boxes to associate with a tag. If the value is not detected by OCR, or you want to redact a picture like profile image, you can *draw a region* to associate the area with the tag.
```

Comment:
suggestion: move `as much as possible` out of the `( )`

Reply:
suggestion: Only label the PII or sensitive fields by following....

I'm not sure about this one, but I think the `only` is to decorate `Label the PII or sensitive fields only` but not `only by labeling`, and putting the `only` in this place create ambiguity.

Reply:
Added a reference link to the Appendix and updated the wording per your suggestion.